src/components/TestComponent.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import propTypes from 'prop-types';
+import connectComponent from '../lib/connect-component';
```

Comment:
Why are you still importing `lib` this way when it has been aliased as `@Lib`?

Reply:
for some reasons it didn't work when I used the alias. But will take a critical look at it and make a new push